juspay/hyperswitch-control-center
# chore: remove duplicate api calls

## Type of Change




## Description
Profile list api is getting called twice on load, this change stops this behaviour
Stopped duplicate re-mounting of Sidebar component to prevent duplicate api calls on switch and reload for merchant and org list apis

## Motivation and Context

preventing unnecessary api calls

## How did you test it?

- reload home page and check for org and merchant user list apis
- switch merchant and check for the same apis
- the duplicate api calls have been removed

## Where to test it?


## Checklist
